Dockerfile_subbridge_e2e

```diff
@@ -0,0 +1,58 @@
+# The Licensed Work is (c) 2022 Sygma
+# SPDX-License-Identifier: LGPL-3.0-only
+
+FROM paritytech/ci-linux:production as builder
+
+WORKDIR /code
+
+COPY ./scripts/phala-subbridge/khala-e2e.config.json .
+
+# build khala-parachain
+RUN git clone https://github.com/Phala-Network/khala-parachain.git ./khala-parachain
+WORKDIR /code/khala-parachain
+RUN git checkout sygma-integration
+RUN cargo build --release --features=all-runtimes
```

Comment:
Run with `cargo build --profile production --features=thala-parachain-runtime` should be more effecient because it will only compile one runtime (thala)

Reply:
that make senses cuz we only use thala runtime lol